packages/core/src/stylable-resolver.ts

```diff
@@ -51,7 +51,7 @@ export class StylableResolver {
         protected requireModule: (modulePath: string) => any,
         protected cache?: StylableResolverCache
     ) {}
-    private getModule({ context, from }: Imported): CachedModule {
+    private getModule({ context, from, request }: Imported): CachedModule {
```

Comment:
what's the from? why is it used for the cache key?

Reply:
from the the abs path via path.resolve which should be still ok for the cache key.

Reply:
I'm not sure how it's ok for the cache key:
`path.resolve(context, './aaa/some-file.js') === path.resolve(context, 'aaa/some-file.js')`

Reply:
and....it's the same file.